addons/account/models/account_move_line.py

```diff
@@ -1220,16 +1220,15 @@ def _check_off_balance(self):
 
     @api.constrains('account_id', 'display_type')
     def _check_payable_receivable(self):
+        if self.env.context.get('winbooks_import'):
+            return
+
         for line in self:
             account_type = line.account_id.account_type
             if line.move_id.is_sale_document(include_receipts=True):
-                if account_type == 'liability_payable':
-                    raise UserError(_("Account %s is of payable type, but is used in a sale operation.", line.account_id.code))
```

Comment:
We shouldn't remove a constraint like this because of an import

Reply:
It's part of the spec :shrug: 

![image](https://github.com/user-attachments/assets/6e756e73-7ba5-4394-8be9-bb7ac6057cc1)

Reply:
But this is not fixing it though. We still only accept payable for payment term lines of vendor bills and receivable for the payment term lines of customer invoices.

That fix only changed the error displayed

Reply:
It's not exactly the same.
Prior to this commit, the constraint would prevent using receivable/payable accounts on *any* line of a vendor bill/customer invoice.
Whereas now, the constraint will *only* apply to `payment_terms` lines (which is less restrictive and the expected behavior, at least for now, according to TSB).

Reply:
But a lot of logic is based on the account being either payable or receivable a bit everywhere... It isn't safe to have a payable/receivable account on non payment terms lines.

Just look at `_compute_display_type` for instance...